src/server/api/routers/waitlist.ts

```
@@ -74,6 +74,21 @@ export const waitlistRouter = createTRPCRouter({
         },
       });
 
+      // Send confirmation email — fire-and-forget, submission succeeds even if email fails
+      const emailResult = await sendTemplateEmail({
+        email,
+        subject: "You're on the ISO waitlist — we'll be in touch",
+        template: "waitlist-confirmation",
+        props: { name: input.name },
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'props' should be before 'template'.

(sort-keys-fix/sort-keys-fix)